examples/spouts/go-kafka-spout/README.md

```diff
@@ -0,0 +1,231 @@
+# Commit messages from a Kafka queue
+
+A simple example using spouts with [Kafka](https://kafka.apache.org) to process messages and write them to files.
+
+## Background
+
+This example connects to a Kafka queue and reads a topic.
+It then writes each message in the topic to a file named by the topic and offset. 
+It uses Kafka group ids to maintain a cursor into the offset in the topic, 
+making it resilient to restarts.
+
+## Introduction
+
+Apache Kafka is a distributed streaming platform.  
+It's used in a variety of applications to provide communications between microservices.  
+Many Pachyderm users use Kafka to ingest data from legacy data sources using Pachyderm spouts.
+
+Pachyderm spouts are a way to ingest data into Pachyderm 
+by having your code get the data from inside a Pachyderm pipeline.
+
+This is the simplest possible implementation of a Pachyderm spout using Kafka to ingest data.
+The data ingested is simply the message posted to Kafka.
+The filename is derived from the Kafka topic name and the message's offset in the topic.
+You should be able to easily adapt it to your needs.
+
+## Setup
+
+This example includes a pre-configured Kafka cluster you can deploy on Amazon EKS,
+adapted from [Craig Johnston's excellent blog post](https://imti.co/kafka-kubernetes/).
+If you'd like to adapt it for your own cluster on GCP, Azure, or on-premises Kubernetes deployment,
+the file 001-storage-class.yaml is probably the only thing you'd need to change.
+You can replace the parameters and provisioner with the appropriate one for your environment.
+
+If you already have a Kafka cluster setup, you may skip step 1 of the Kafka setup.
```

Comment:
```suggestion
If you already have a Kafka cluster setup, you can skip the Kafka Setup step and proceed to [Pachyderm setup](#pachyderm-setup).
```

Reply:
In this case, it really is just step 1 of the Kafka setup they should skip.  Steps 2 and beyond of the Kafka portion involve creating the topic you want to use, adding messages to the topic, and testing to see if messages were sent.